plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/gramine/GenerateGramineBundle.kt

```diff
@@ -154,12 +155,15 @@ open class GenerateGramineBundle @Inject constructor(
      * This will create a .manifest.sgx and a .sig files into the output dir.
      */
     private fun generateSgxManifestAndSigstruct() {
+        signingKey.copyToOutputDir(SIGNING_KEY)
```

Comment:
Can't you reuse `prepareFile` in `LinuxExec`? It does copy file into the temp dir within the project. Not sure what location is required when creating the manifest. You also have the `cleanPreparedFiles` to delete the file.

Reply:
It seems to work with your suggested approach too
```
Starting process 'command 'docker''. Working directory: /home/marco/conclave-tutorials/hello-world/enclave Command: docker run --rm -u 1000:1000 -v /home/marco/conclave-tutorials/hello-world/enclave:/project -w /project/build/conclave/debug/gramine-bundle conclave-build:latest gramine-sgx-sign --manifest=java.manifest --key=/project/.linuxexec/sample_private_key14535025473192576462pem --output=java.manifest.sgx
Successfully started process 'command 'docker''
```